### PR TITLE
New items

### PR DESCRIPTION
**Pull request by Darxeal**

### CODE REVIEW

File: rlbot_gui/gui/csv/items.csv [hunk 1]
```diff
@@ -2912,66 +2938,169 @@
...
+5438,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_MuscleCar2,?INT?Products.ShopItem_Body_MuscleCar2.Label?
+5439,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_O2,?INT?Products.ShopItem_Body_O2.Label?
+5440,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Pumpernickel,?INT?Products.ShopItem_Body_Pumpernickel.Label?
+5441,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Rhino2,?INT?Products.ShopItem_Body_Rhino2.Label?
+5442,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Sourdough,?INT?Products.ShopItem_Body_Sourdough.Label?
+5443,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Takumi_II,?INT?Products.ShopItem_Body_Takumi_II.Label?
+5444,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Torch2,?INT?Products.ShopItem_Body_Torch2.Label?
+5445,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Twinzer,?INT?Products.ShopItem_Body_Twinzer.Label?
+5446,ShopItem,Product_TA ProductsDB.Products.ShopItem_Body_Wheatbread,?INT?Products.ShopItem_Body_Wheatbread.Label?
+5617,Skin,Product_TA ProductsDB.Products.skin_octane_elp,Octane: Pel�
```
Comment (tarehart) [inline rlbot_gui/gui/csv/items.csv:3104]: I noticed the �, can this be replaced with the right character, whatever that is?

**Review by tarehart — APPROVED** [review]
Thanks Darxeal!

File: rlbot_gui/gui/csv/items.csv (hunk 1 above)
Comment (Darxeal) [reply rlbot_gui/gui/csv/items.csv:3104]: There are already multiple items with those. We would have to manually fix those every time we re-generate the file. Yes, you would only have to discard the changes, but sometimes the new items are spread all over the place so you'd have to manually search for them. But maybe there's a way to automate that?

Comment (Darxeal) [reply rlbot_gui/gui/csv/items.csv:3104]: I noticed that the github csv preview shows the character correctly. Maybe we just have to change the file encoding or something, I'll investigate

**Comment by tarehart** [comment]
Nice!

**Pull request merged**